### ISSUE TITLE
Add vocabulary for Redux

### ISSUE THREAD

**Issue by ChenyunZhang** [sylwiavargas/The-Code-Bending-Dictionary#51]
hey @sylwiavargas could we add also vocabulary for Redux? Could I go ahead and add Redux words to seeds?

**Comment by sylwiavargas** [comment]
hey @ChenyunZhang ! Thank you for this idea — I've assigned you to this issue ✨

**Comment by sylwiavargas** [comment]
hey @ChenyunZhang , would you still like to add vocab for Redux? If not — no problem! If yes, would you add definitions as well?

**Comment by ChenyunZhang** [comment]
Yes, I will start work on it. I will add the definitions as well.
